android/KMEA/app/src/main/java/com/tavultesoft/kmea/LanguageListActivity.java

```diff
@@ -190,13 +190,6 @@ public void onBackPressed() {
     finish();
   }
 
-  @Override
-  protected void onActivityResult(int requestCode, int resultCode, Intent data) {
-    if (resultCode == 1) {
-      finish();
-    }
-  }
-
```

Comment:
These were removed since @safolit pointed out requestCode is never 1 when transitioning back from the child activity